content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,119 +2,152 @@
 Getting started with Helpdesk
 =============================
 
-Odoo :guilabel:`Helpdesk` is a ticket-based customer support application. Teams can track,
-prioritize, and solve customer issues from their pipeline, which is organized in customizable
-stages. Multiple teams can be configured and managed in one dashboard.
+Odoo *Helpdesk* is a ticketing-based customer support application. Multiple teams can be configured
+and managed in one dashboard, each with their own pipeline for customer submitted tickets. Pipelines
+are organized in customizable stages that enable teams to track, prioritize, and solve customer
+issues quickly and efficiently.
 
-Create Helpdesk teams
-=====================
+Create a Helpdesk team
+======================
 
-Setting up multiple teams allows for tickets to be grouped by location or by support type.
-
-To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`. To
-create a new team, click on the :guilabel:`New` button in the top left of the dashboard. From there,
-name the new team, and fill out the remaining fields as defined in the following sections on the
-form.
+To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams`. To create a new team, click on the :guilabel:`New` button in the top-left of the
+dashboard.
 
 .. image:: getting_started/helpdesk-teams-list.png
    :align: center
-   :alt: View of the Helpdesk teams page in Odoo Helpdesk
+   :alt: View of the Helpdesk teams page in Odoo Helpdesk.
+
+Enter a :guilabel:`Name` for the new team. Enter a description of the team to the field below the
+team name, if desired.
+
+.. important::
+   This description is published on the public facing :doc:`website form <receiving_tickets>` where
+   customers and portal users submit tickets. The description included in this field should **not**
+   include any information that is for internal use only.
```

Comment:
You might want add that the description appears in the 'About Our Team' box on the right, on the web. I  looked at the image and was not 100% sure if that is where the description is- but if so, you might want to add that to make it crystal clear. Or you can also add a red box around where the description is to connect the dots for the reader. Or both!

Reply:
I don't necessarily want to overly define the box/location on the webpage since it is editable and customizable, so it can change. (As in, I don't want to say "it appears as an 'about the team' box" since they can make it say whatever they want). Since there really isn't anything else in the screenshot that could be referring to the team description, I don't think it needs a lot of additional clarification.

Reply:
I forgot they can change how it looks! I assumed that was how it was, full stop. I agree with your reasoning then- thanks for clearing that up!